src/tools/rbtree.c

```diff
@@ -1216,6 +1238,7 @@ void rbtree_print(const rbtree_t *tree) {
         return;
     }
     unsigned bdepth = 0;
+    cache_check(tree);
```

Comment:
Why did you place the call to `cache_check` here? Move it prior to `unsigned bdepth` declaration.